parser/v2/forexpressionparser_test.go

```diff
@@ -57,6 +57,7 @@ func TestForExpressionParser(t *testing.T) {
 								},
 							},
 						},
+						LineBreak: true,
```

Comment:
This change needs to be made in a lot of places. Would it make more sense to invert it, so that it's "NoTrailingLineBreak"? That way, it would be false by default.

Reply:
Personally I never wrote an property with a negative sign but since In go I can't say that Element is by default true. That better.

After maybe I can get rid of this `LineBreak` property